_includes/ee-linux-install-reuse.md

```diff
@@ -121,7 +121,7 @@ You can install Docker EE in different ways, depending on your needs:
     ```bash
     $ sudo yum-config-manager \
         --add-repo \
-        "$DOCKERURL/{{ linux-dist-url-slug }}/docker-ee.repo"
+        $(cat /etc/yum/vars/dockerurl)/docker-ee.repo"
```

Comment:
I think my way is better now. :)

Reply:
OK, here's a longer explanation why I added this.

The `$DOCKERURL` is something you get from a web page, and during step 2 you save it to a file. The steps are:
 - copy (or type) the command from step 2 from docs to a terminal, without pressing Enter
 - copy the EE URL repo from Docker EE registration page
 - remove $DOCKERURL from the command, paste the above-copied URL instead
 - make sure everything looks good, press Enter.

Now, the same sequence is needed during step 6 here, but it can be avoided as we already saved the URL to a file and we can reuse it right here. This is what this hunk does, hope my explanation makes sense.

Reply:
Your explanation makes sense but the steps in the published file are no longer the same as what you outline.

Reply:
Oh, now I see. Yeah, makes total sense. Will remove this hunk.